src/hiero_sdk_python/query/transaction_record_query.py

```
@@ -16,13 +16,29 @@ class TransactionRecordQuery(Query):
     Represents a query for a transaction record on the Hedera network.
     """
 
-    def __init__(self, transaction_id: Optional[TransactionId] = None):
+    def __init__(self, transaction_id: Optional[TransactionId] = None, include_duplicates: bool = False):
```

Ruff (0.14.14)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)